vhdbuilder/packer/components.json

```diff
@@ -72,14 +72,16 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v1.3.4",
-        "v1.3.4-1"
+        "v1.3.4-1",
```

Comment:
we are only adding, not retiring older versions. 
our VHDs are at capacity and we are having build issues, can we please retire older versions if we are adding newer ones?

Reply:
if old image is still used in aks-rp, we should keep it until new image is used in aks-rp

Reply:
then you should remove `v1.3.4` also since it's not used any more

```
{{- else if eq .component "secrets-store-csi-driver" -}}
  {{- if semverCompare ">=1.24.0" .version -}}v1.3.4-1
  {{- else -}}v1.3.0.3
  {{- end -}}
```